samples/tcpsrv.lua

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/env lua
+
+--
+-- Sample TCP echo server. Listens on 3333 port
+-- Use netcat to send strings to server (ncat 127.0.0.1 3333)
+--
+
+local lgi = require 'lgi'
+local Gio = lgi.Gio
+
+local app = Gio.Application{application_id = 'org.test.tcptest',
+                            flags = Gio.ApplicationFlags.NON_UNIQUE}
+
+local service = Gio.SocketService.new()
+service:add_address(
+   Gio.InetSocketAddress.new_from_string("127.0.0.1", 3333),
+   Gio.SocketType.STREAM,
+   Gio.SocketProtocol.TCP)
+
+local function get_message(conn, istream, ostream)
```

Comment:
```suggestion
local function do_echo_connection(conn, istream, ostream)
```
(or something like that, "get message" is not what the function is doing, I think)